### ISSUE TITLE
[BUG]: 为什么改变了group norm的组数 从32 到16？

### ISSUE THREAD

**Issue by guohe369** [hpcaitech/ColossalAI#3109]
### 🐛 Describe the bug

我注意到stable diffusion v2源码的group数是32，在本仓库的example中实现的stable diffusion v2却被改成了16

![image](https://user-images.githubusercontent.com/45527236/224474775-2b3e16e6-0dad-4bd1-b961-1768c5a0b0b7.png)

231 行

def normalization(channels):
    """
    Make a standard normalization layer.
    :param channels: number of input channels.
    :return: an nn.Module for normalization.
    """
    return nn.GroupNorm(16, channels)
    # return GroupNorm32(32, channels)

请问这是为什么呢？

**Comment by binmakeswell** [comment]
Hi @guohe369 We have updated a lot. Please check the latest code. This issue was closed due to inactivity. Thanks.
